.github/workflows/deploy-2-start.yml

```diff
@@ -101,7 +101,7 @@ jobs:
 
           # Update spack-packages
           git -C ${{ steps.path.outputs.spack-packages }} fetch
-          git -C ${{ steps.path.outputs.spack-packages }} checkout --force ${{ steps.versions.outputs.packages }}
+          git -C ${{ steps.path.outputs.spack-packages }} reset --hard origin/${{ steps.versions.outputs.packages }}
```

Comment:
Have you already switched to the branch? The steps I do are:
1) Fetch
2) Be in the correct branch
3) Sync current branch to origin's branch

Reply:
Good point. For some reason I thought `git reset` just moved the `HEAD`, not the local branch. Fixed in https://github.com/ACCESS-NRI/build-cd/pull/166/commits/c3b20321f2dadbdaf052836db58387cd568bdb82